web/src/OverviewActionBar.tsx

```diff
@@ -475,6 +478,7 @@ export default function OverviewActionBar(props: OverviewActionBarProps) {
           filterSet={props.filterSet}
           alerts={alerts}
         />
+        {isSnapshot || <ClearLogs resource={resource} />}
```

Comment:
will this make the text "true" appear in snapshots? not sure that's what you want

Reply:
No, this is a goofy but common abuse of JS logical operators -> https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator - `false || expression` evaluates to `expression`, not a `boolean` type 😬

(JSX explicitly ignores `false` from an expression within element content to enable this use case)

If there's a different pattern used throughout Tilt UI, I can swap. Is `isSnapshot ? null : <ClearLogs ... />` preferable?